server/models/ConfigModel/index.js

```diff
@@ -1,22 +1,80 @@
+import { ObjectId } from 'mongodb'
 import defaultUserConfig from '../../configs/defaultUserConfig'
 import { collections } from '../../utils/mongoCollections'
 
 const ConfigModel = {
+  destroy: async (db, configId) => {
+    return await db
+      .collection(collections.configs)
+      .deleteOne({ _id: new ObjectId(configId) })
+  },
+  edit: async (db, configId, configAttributes) => {
+    console.log(configId, configAttributes)
+    return await db
+      .collection(collections.configs)
+      .findOneAndUpdate(
+        { _id: new ObjectId(configId) },
+        { $set: configAttributes },
+        { returnOriginal: false }
+      )
+  },
+  index: async (db, userId) => {
+    return await db
+      .collection(collections.configs)
+      .aggregate(ConfigModel.loadAllConfigurationsMongoQuery(userId))
+      .toArray()
+  },
+  loadAllConfigurationsMongoQuery: (userId) => {
+    const users = 'users'
+    const $owner = '$owner'
+    const $uid = '$uid'
+    const $$owner = '$$owner'
+    const resultPropertyName = 'ownerUser'
+    const $ownerUser = '$ownerUser'
+    const $display_name = '$display_name'
+
+    return [
+      { $match: { readers: userId } },
+      {
+        $lookup: {
+          from: users,
+          let: { owner: $owner },
+          pipeline: [
+            {
+              $match: {
+                $expr: { $eq: [$$owner, $uid] },
+              },
+            },
+            {
+              $project: {
+                icon: 1,
+                uid: 1,
+                name: $display_name,
+                _id: 0,
+              },
+            },
+          ],
+          as: resultPropertyName,
+        },
+      },
+      { $unwind: $ownerUser },
+    ]
+  },
+  save: async (db, configAttributes) => {
+    const config = ConfigModel.withDefaults(configAttributes)
+
+    return await db.collection(collections.configs).insertOne(config)
+  },
+  show: async (db, userId) => {
```

Comment:
this method can be `find` to get the user from the DB, the controller action is `show`.

Reply:
I think show makes sense or findOne. Mongo has a find method and that is a method that finds all documents that match a criteria, I think it would confuse developers